README.md

```diff
@@ -14,11 +14,11 @@ Swift SDK for the [Contentful][1] Content Delivery API.
 
 [Contentful][1] provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
 
-However, it provides a more usable API than the [Objective-C SDK][4] and has support for more API features. It is recommended to use contentful.swift over contentful.objc as future development at Contentful will focus on Swift rather than Objective-C.
+It is recommended to use [contentful.swift][10] over [contentful.objc][4] as future development at Contentful will focus on Swift.
```

Comment:
maybe even emphasize or bold this statement?

Reply:
Mm, I think the more important thing is that it visually stands out on the contentful.objc repo (which I believe it does already). I think if people end up at contentful.swift, they likely want to use swift anyway. If you feel strongly that I should add more emphasis here though, just let me know ;-)